va/va.h

```diff
@@ -1130,6 +1130,50 @@ VAStatus vaDestroyContext (
     VAContextID context
 );
 
+//Multi-frame context
+typedef VAGenericID VAMFContextID;
+/**
+ * vaCreateMFContext - Create a multi-frame context
+ *  Multi-frame context allow to run tasks from different
+ *  contexts in single batch buffer for performance optimization.
+ *  Multi-frame context is destroyed through vaDestroyContext
+ *  dpy: display
+ *  mf_context: created multi-frame context id upon return
+ */
+VAStatus vaCreateMFContext (
+    VADisplay dpy,
+    VAMFContextID *mf_context    /* out */
+);
+
+/**
+ * vaMFAddContext - Add context into multi-frame and cerate
+ *  association with multi-frame context.
+ *  After association is done, current context will not submit 
+ *  render targets during vaEndPicture, but during vaMFSubmit.
+ *  dpy: display
+ *  context: VAContextID to be added
+ *  mf_context: VAMFContextID where context is added
+ */
+VAStatus vaMFAddContext (
+    VADisplay dpy,
+    VAContextID context,
+    VAMFContextID mfe_context
```

Comment:
please replace mfe_context with mf_context in this commit, we should use the same name style.

Reply:
FIxed